js/widgets/fixedToolbar.js

```diff
@@ -37,59 +39,54 @@
 
 return $.widget( "mobile.toolbar", $.mobile.toolbar, {
 	options: {
-		position: null,
+		position:null,
 		visibleOnPageShow: true,
 		disablePageZoom: true,
-		transition: "slide", //can be none, fade, slide (slide maps to slideup or slidedown)
+
+		// Can be none, fade, slide (slide maps to slideup or slidedown)
+		transition: "slide",
```

Comment:
@jaspermdegroot there is the hide method on fixed toolbar so I think it should still live here in the fixedToolbar extension